search-app/src/cmr/search/services/parameters/conversion.clj

```diff
@@ -469,6 +470,22 @@
           (cqm/map->MissingCondition {:field :collection-data-type})])
         (cqm/string-condition :collection-data-type value case-sensitive pattern)))))
 
+;; Converts collection progress parameter and values into conditions.
+;; Supports OR (default) and AND operations for multiple values.
+(defmethod common-params/parameter->condition :collection-progress
+  [context concept-type param value options]
+  (if (sequential? value)
+    ;; Handle multiple values with AND or OR logic
+    (if (= "true" (get-in options [param :and]))
```

Comment:
I can't remember if the value is normalized to lower case for you at this point. If so, this looks great.